From aabf7692ce50d3ab3a01bb334ed38cacfd909fad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Otto=20Kek=C3=A4l=C3=A4inen?= <otto@debian.org>
Date: Thu, 10 Oct 2024 17:05:07 -0700
Subject: [PATCH] Add an example of Vsc-* usage to make section easy to
 understand

When presenting the Vcs-Browser and Vcs-<type> fields, start by showing
one typical example. With the illustration visible it is much easier for
a new maintainer to read and grasp the technical definition on what all
syntax is allowed and what it means.
---
 policy/ch-controlfields.rst | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/policy/ch-controlfields.rst b/policy/ch-controlfields.rst
index 530755a..eb673ff 100644
--- a/policy/ch-controlfields.rst
+++ b/policy/ch-controlfields.rst
@@ -1080,6 +1080,14 @@ repository where the Debian source package is developed.
 ``Vcs-Browser``
     URL of a web interface for browsing the repository.
 
+    The example below illustrates the difference between the browsing location and actual VCS
+    location:
+
+    .. code-block:: debcontrol
+
+        Vcs-Browser: https://salsa.debian.org/debian/grep
+        Vcs-Git: https://salsa.debian.org/debian/grep.git
+
 ``Vcs-<type>``
     The field name identifies the VCS. The field's value uses the version
     control system's conventional syntax for describing repository
-- 
2.25.1

